site/docs/v1/tech/premium-features/scim/azure-ad.adoc

```diff
@@ -0,0 +1,34 @@
+---
+layout: doc
+title: Azure AD SCIM Client
+description: Configure Azure AD as a SCIM client
+navcategory: premium
+---
+
+:sectnumlevels: 0
+
+== Overview
+
+This is a basic example to get started configuring Azure AD as a SCIM client and establish a connection between FusionAuth and Azure AD for provisioning purposes.
+
+For complete documentation, please refer to the Azure AD provisioning documentation.
+
+To configure the integration, in the Microsoft Azure console, navigate to *Azure AD* and click on *Enterprise Applications* and either create a new application, or select the application you will be using for SCIM provisioning. Once you have selected the application, click on *Provisioning* in the left navigation under the *Manage* section.
+
+Here you will find an *Update Credentials* option in the *Manage Provisioning* section at the bottom of the page. Expand the *Admin Credentials* section find the required configuration fields.
+
+.[breadcrumb]#Home -> Azure AD -> Enterprise Applications -> Provisioning -> Update credentials#
+image::azure-ad-scim-client-credentials.png[Azure AD SCIM Provisioning Integration,width=1200,role=shadowed]
```

Comment:
We don't need role=shadowed, do we? I thought that was deprecated.

Reply:
I thought it still did something, but I'll confirm and remove if it is dead. Thanks.